### ISSUE TITLE
Contrib Docker prod templates creates a folder called ./wa/letsencrypt/acme.json/

### ISSUE THREAD

**Issue by timdonovanuk** [thecodingmachine/workadventure#2062]
You end up with a folder, not a file, called wa/letsencrypt/acme.json/...

![image](https://user-images.githubusercontent.com/8156439/162591452-b8d8e43b-6b7b-43a4-a571-5506aa6c0c97.png)

This breaks the entire reverse-proxy traefik piece.

**Comment by gparant** [comment]
Hi @timdonovanuk, if you're using ACME traefik to create a certificate with the let's encrypt TLS challenge to use https, you must already create file acme.json and change the rights.

So you can stop services, delete the folder and apply this commands: 
```
touch wa/letsencrypt/acme.json
chmod 600 wa/letsencrypt/acme.json
```

**Comment by lukashass** [comment]
This should have been fixed by #2052 

Make sure to be on the latest `develop` branch version and delete the directory before trying again :wink:

**Comment by moufmouf** [comment]
Closed by #2052, thanks @lukashass !
